source/_posts/2017-09-01-gsoc-2017-data-visualization-using-daru-view.markdown

```diff
@@ -410,7 +431,7 @@ as data source parameter.
 ## Future Work
 
 daru-view will be more powerful and simple in near future. Developers can add
-more libraris in daru-view easily, if required. To add library follow the setups given in 
+more libraris in daru-view easily, if required. To add library follow the setups given in
```

Comment:
Typo: libraries

Reply:
Just run the whole thing through a spell checker?

Reply:
Yes, spell checking done. Hopefully now it is okay.